### PR TITLE
ci: make ubuntu versions consistent across workflow file jobs

### PR DESCRIPTION
**Pull request by jhlegarreta**
- ci: make ubuntu versions consistent across workflow file jobs
- ci: use GitHub Action to free up disk space

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Cross-referencing https://github.com/InsightSoftwareConsortium/itk-wasm/pull/907#issuecomment-1689780836.

Notes about the commit message:
- Used [commitizen](https://github.com/commitizen-tools/commitizen) to create the commit using `cz commit`. Hope this is the correct way.
- `cz commit` recommended to keep the subject to all lowercases. Let me know if we follow that here.
- `cz commit` does not seem to enforce a new line at e.g. 72 characters. Let me know if a line length limit is used here, and if so, how to configure that.

Thanks.

**Review by thewtex — APPROVED** [review]

**Review by thewtex — CHANGES_REQUESTED** [review]
CI catches updates needed in the "Free up disk space" sections of the CI configurations

**Comment by thewtex** [comment]
Thank you Jon Haitz @jhlegarreta !

It looks like the clean-up disk space steps need to be updated for Ubuntu 22.04.


> Used commitizen to create the commit using cz commit. Hope this is the correct way.

That works.

> cz commit recommended to keep the subject to all lowercases. Let me know if we follow that here.

Currently we support all lowercase but also starting the subject with a capital, e.g. `ci: make ubuntu versions consistent` or `ci: Make ubuntu versions consistent`.

> cz commit does not seem to enforce a new line at e.g. 72 characters. Let me know if a line length limit is used here, and if so, how to configure that.

There is a line length of 300, configured here: https://github.com/InsightSoftwareConsortium/itk-wasm/blob/b1289e36d0b2483c8687d9325fb16ed604be4b15/package.json#L242-L245

for pasting long error messages. CI will fail if that is exceeded.

**Comment by jhlegarreta** [comment]
> It looks like the clean-up disk space steps need to be updated for Ubuntu 22.04.

See if 0558703 makes the trick.

> Currently we support all lowercase but also starting the subject with a capital, e.g. ci: make ubuntu versions consistent or ci: Make ubuntu versions consistent.

OK.

> There is a line length of 300, configured here:

OK.

**Comment by jhlegarreta** [comment]
> See if 0558703 makes the trick.

The disk space issue is still not solved:
https://github.com/InsightSoftwareConsortium/itk-wasm/actions/runs/6229582807/job/16908321942?pr=934

**Comment by thewtex** [comment]
Perhaps an `apt-get update` is needed to update mirror contents?

**Comment by thewtex** [comment]
Seems to working in the others, but the `Javascript, Typescript` looks to need updates.

**Comment by jhlegarreta** [comment]
Reworded my commit message to honor its changes, as freeing disk space is now addressed in a separate commit.

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -206,20 +159,14 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: 'Free up disk space'
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
+
+    - name: Install cypress deps
```
Comment (jhlegarreta) [inline .github/workflows/examples.yml:165]: Should this be put into a separate commit/PR ? I'd say that some tests were failing after this addition.

Comment (thewtex) [reply .github/workflows/examples.yml:165]: Yes, I added another commit to try to get this green for integration.

**Review by thewtex — APPROVED** [review]
Thanks again @jhlegarreta for your awesome work on this!

The remaining failing CI check was addressed in `main` by https://github.com/InsightSoftwareConsortium/itk-wasm/commit/8a9925b3647993e0074d486877bd02492c4f5c93

**Pull request merged**